server/package.json

```
@@ -65,7 +65,7 @@
   "scripts": {
     "start": "ts-node Server.ts",
     "start-dev": "nodemon Server.ts",
-    "build": "yarn remove-dist && tsc -b",
+    "build": "yarn remove-dist && tsc -b && cp worker/tasks/*.sh worker/dist",
```

I believe it should be 
```suggestion
    "build": "yarn remove-dist && tsc -b && cp worker/tasks/*.sh worker/dist/tasks",
```

---

No, it's actually OK like this:

https://github.com/regen-network/regen-server/blob/0b5a9a4d1be78334e2f9d24ec8caaf0ebba55838/server/worker/worker.ts#L11